sql/indicator_functions.sql

```diff
@@ -280,7 +280,42 @@ CREATE FUNCTION PECG_Indicator17(
     p_startAge INT(11),
     p_endAge INT (11),
     p_includeEndAge TINYINT(1),
-    p_gender VARCHAR(1)) RETURNS INT(11)
+    p_gender VARCHAR(1),
+    p_monthOffset INT(11)) RETURNS INT(11)
```

Comment:
@IipperMangesh the parameter `p_monthOffset` is not needed for the function `PECG_Indicator17` (as it is hardcoded when calling patientDidntCollectARV and patientHasScheduledAnARTAppointment)